semantic/src/semantic.less

```diff
@@ -63,4 +62,4 @@
 & { @import "definitions/modules/sidebar"; }
 & { @import "definitions/modules/sticky"; }
 & { @import "definitions/modules/tab"; }
-& { @import "definitions/modules/transition"; }
+& { @import "definitions/modules/transition"; }
```

Comment:
Do we need that newline GitHub is complaining about?

Reply:
Having a newline at the end helps with diffs when content is appended, but in this case all the changes in the semantic directory is just reverting back to what was there initially as the changes resulted in some visual bugs.